### DISCUSSION TITLE
Make Vertical the Default Tiling Direction?

### DISCUSSION THREAD

**Discussion by Wakers** [glzr-io/glazewm#574] [Q&A]
Can the config file be modified in such a way?

Or can I set one workspace to tile everything vertically and the other to tile everything horizontally?

**Comment by fmos** [comment]
You can achieve this by putting `'toggle-tiling'` in the `startup_commands` in your `config.yaml`, for example:

```
general:
  startup_commands: [
    'shell-exec zebar startup',
    'toggle-tiling'
  ]
  # ...
```
